### ISSUE TITLE
Missing InAppMessagingDisplayResources.bundle with Pod installation 

### ISSUE THREAD

**Issue by nikhildange** [firebase/firebase-ios-sdk#3090]
Xcode version: 10.2
Firebase SDK version: 5.19.0
Firebase Component:  InAppMessagingDisplay 
    - Firebase/CoreOnly - FirebaseInAppMessagingDisplay (= 0.13.1)
Component version: 5.19.0

Using pod to include firebase in my app.

_App crashes when trying to present InApp display message with:_
5.19.0 - [Firebase/InAppMessaging][I-IAM160001] There are 1 test messages and 0 regular messages and 0 Firebase Analytics events to watch after resetting the message cache
5.19.0 - [Firebase/InAppMessaging][I-IAM240002] Fetch is done. Start message rendering flow.
5.19.0 - [Firebase/InAppMessaging][I-IAM160007] Returning a test message for app foreground display
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSBundle initWithURL:]: nil URL argument'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010b2936fb __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x000000010a2b6ac5 objc_exception_throw + 48
	2   CoreFoundation                      0x000000010b293555 +[NSException raise:format:] + 197
	3   Foundation                          0x000000010786a475 -[NSBundle initWithURL:] + 87
	4   Foundation                          0x000000010786a524 +[NSBundle bundleWithURL:] + 45

It cannot find the InAppMessagingDisplayResources.bundle included with the framework.

**Comment by maksymmalyhin** [comment]
@nikhildange Thank you for bringing this up. I wonder, if it is possible for you to try the lates Firebase version 6.1.0 and check if the issue is still reproducible. If yes, could you please share `Podfile` and `Podfile.lock` or ideally a test project to reproduce the issue.

**Comment by nikhildange** [comment]
Hi @maksymmalyhin, Thanks for replying. I have updated pod to the latest version, still facing issue.
- Firebase/InAppMessagingDisplay (6.1.0):
    - Firebase/CoreOnly
    - FirebaseInAppMessagingDisplay (~> 0.14.0)

As you asked for I have attached them in .txt extension as this doesn't support uploading .lock or podfile files.
[Podfile.lock.txt](https://github.com/firebase/firebase-ios-sdk/files/3225761/Podfile.lock.txt)
[Podfile.txt](https://github.com/firebase/firebase-ios-sdk/files/3225762/Podfile.txt)

**Comment by maksymmalyhin** [comment]
@nikhildange Thank you for the file. I've tried the following test app using the provided Podfile(.lock). I could successfully receive a test message on a Simulator and didn't get the crash. Would you please share the missing steps or project settings to reproduce the issue.
[TestApp.zip](https://github.com/firebase/firebase-ios-sdk/files/3230334/TestApp.zip)

**Comment by nikhildange** [comment]
@maksymmalyhin There are no as such step to miss How do I share project settings to help you reproduce?

**Comment by paulb777** [comment]
You could try the FIAM quickstart - https://github.com/firebase/quickstart-ios/tree/master/inappmessaging

**Comment by nikhildange** [comment]
Issue Resolved

What i did?
I started debugging app by placing breakpoint where path for the resource was nil. when you hit a breakpoint, the value were nil & showed _proj_name was compiled with optimization - stepping may behave oddly_ message, preventing you from watching real variable values because the in pod project setting optimization was different from main & target setting, so changed to none.

![image](https://user-images.githubusercontent.com/11014775/58580294-5e322680-826a-11e9-976e-c201dc3fa2c2.png)

Now I was able to see value/path for bundle, but InAppMessagingDisplayResources.bundle was still missing from proj_name.app.
